admin-dev/themes/new-theme/js/pages/catalog/product/specific-price-form-handler.ts

```diff
@@ -448,6 +455,21 @@ class SpecificPriceFormHandler {
     }
   }
 
+  /**
+   * @param boolean usePrefixForCreate
+   *
+   * @private
+   */
+  private changeCurrencySymbol(
+    usePrefixForCreate: boolean,
+  ): void {
+    const selectorPrefix = this.getPrefixSelector(usePrefixForCreate);
+    const currencySymbol = $(`${selectorPrefix}sp_id_currency option:selected`).data('currency-symbol');
+
+    $(`${selectorPrefix}sp_price`).parent().find('.input-group-append .input-group-text').text(currencySymbol);
+    $(`${selectorPrefix}sp_reduction_type option[value="amount"]`).text(currencySymbol);
```

Comment:
do not hesitate to add new classes or id in the views.

Reply:
I am not sure that I can add id or class to this element that contain currency symbol :

`src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/bootstrap_4_layout.html.twig`
```html 
{% block money_widget -%}
    <div class="input-group money-type">
        {% set prepend = '{{' == money_pattern[0:2] %}
        {% if not prepend %}
            <div class="input-group-prepend">
                <span class="input-group-text">{{ money_pattern|replace({ '{{ widget }}':''}) }}</span>
            </div>
        {% endif %}
        {{- block('form_widget_simple') -}}
        {% if prepend %}
            <div class="input-group-append">
                <span class="input-group-text">{{ money_pattern|replace({ '{{ widget }}':''}) }}</span>
            </div>
        {% endif %}
    </div>
{%- endblock money_widget %}
```

Do you have any idea or suggestion ? :)

Reply:
I would add a `.js-currency` class to input-group-text, so this selector is used only for the JS and peoples know that it can't be removed

Maybe the same workflow for the `option` element